src/Client/Widgets/NumeralRange.fs

```diff
@@ -0,0 +1,60 @@
+﻿module Client.Widgets.NumeralRange
+
+open Browser.Types
+open Fable.Core.JsInterop
+open Fable.React
+open Fable.React.Props
+
+open Client
+open Common.Numerals
+
+let translateNumeralRange = function
+    | "From0To100" -> Range.From0To100
+    | "From100To1000" -> Range.From100To1000
+    | "From1000To1000000" -> Range.From1000To1000000
+    | "From1000000" -> Range.From1000000
+    | x -> sprintf "Unknown numeral range %s" x |> failwith 
```

Comment:
Maybe worth using dictionary instead to avoid explicit exception code:
```fsharp
let translate s = 
    let translations = 
        dict [ "From0To100", Range.From0To100
                 ...                          ]

    translations.[s]
```
Maybe it's more readable, on you though